sql/2021/ecommerce/all_categories.sql

```diff
@@ -0,0 +1,8 @@
+#standardSQL
+# List of all categories for 2021_07_01
```

Comment:
Ok to remove this query from the PR if the results aren't reader-facing and more informational to the team.